code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -114,6 +114,8 @@ var/list/special_fruits = list()
 	if(seed.juicy)
 		splat_decal(get_turf(hit_atom))
 		splat_reagent_reaction(get_turf(hit_atom),user)
+		var/splasharea = (seed.juicy*round((seed.potency/100), 1)) //2 at 200 potency
```

Comment:
```suggestion
		var/splasharea = (seed.juicy*ceil(seed.potency/100) //2 at 200 potency
```
I recommend using the ceiling instead otherwise any potency under 100 won't cause any splashing at all